src/Configuration.php

```diff
@@ -307,6 +326,16 @@ public function getBasePath(): string
         return $this->basePath;
     }
 
+    public function getComposerJson(): ?string
```

Comment:
This could be extracted in a separate PR and tests added about the values gotten from those, e.g. when the file does not exists, is a directory and so on